internal/kms/vault_sa.go

```diff
@@ -303,6 +307,27 @@ func (kms *vaultTenantSA) getToken() (string, error) {
 		}
 	}
 
+	version, err := c.ServerVersion()
```

Comment:
move this to a helper function, one that can always be called. If the version does not need to create a token, just return no error.